components/neeva/renderer/content_filtering_agent.cc

```diff
@@ -51,11 +51,19 @@ std::unique_ptr<base::MemoryMappedFile> MapRegion(
 ContentFilteringAgent::ContentFilteringAgent(
     blink::ThreadSafeBrowserInterfaceBrokerProxy* broker)
     : task_runner_(base::SequencedTaskRunnerHandle::Get()) {
+
+  // Binds the service as a remote and creates a pipe to the Browser.
+  // See ContentFilteringService::AddInterface() for the handling of the reciever part. 
+  // @Darin why is the Browser broker when registering the Renderer service pipe??
   broker->GetInterface(service_.BindNewPipeAndPassReceiver());
 
+  // Allows the ContentFilterConfig to call OnReceiveNewRules() and 
+  // for the service to "receive" those callbacks. 
   service_->AddRulesListener(receiver_.BindNewPipeAndPassRemote());
 }
 
+// Creates ContentFilter objects. 
```

Comment:
I'd delete this comment block. i don't think it is that helpful.

Reply:
I guess it is self-evident that a throttle == ContentFilter